client/src/components/DataFiles/DataFilesModals/DataFilesModals.scss

```
@@ -9,8 +9,10 @@
   .h-100 {
     padding: 0 10px;
   }
+  .breadcrumbs-container {
+    font-size: 1.2rem;
```

Ugh, I said `1.2rem`. You obliged, but my math was wrong.[^1]

```suggestion
    font-size: 0.75rem;
```

[^1]: 1.2rem would equal 12px if `html`/`:root` font size was different, but I never fixed that for portal. Sigh.